app/services/big_blue_button_api.rb

```diff
@@ -40,6 +40,11 @@ def publish_recordings(record_ids:, publish:)
     bbb_server.publish_recordings(record_ids, publish)
   end
 
+  def update_recordings(record_id:, meta_hash:)
+    filtered_meta = meta_hash.slice :'meta_recording-name'
```

Comment:
What's the purpose of this?

Reply:
Thought for some reason that I should filter out any meta data other than `recording-name``but it's useless because we'll be filtering the params.
I'll remove it.